lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -751,11 +751,44 @@ def password_field(name, attributes = {})
         #   #    <option value="it" selected="selected">Italy</option>
         #   #    <option value="us">United States</option>
         #   #  </select>
+        #
+        # @example Include blank option
+        #   <%=
+        #     # ...
+        #     values = Hash['it' => 'Italy', 'us' => 'United States']
+        #     select :stores, values, options: {include_blank: true}
+        #   %>
+        #
+        #   # Output:
+        #   #  <select name="book[store]" id="book-store">
+        #   #    <option></option>
+        #   #    <option value="it">Italy</option>
+        #   #    <option value="us">United States</option>
+        #   #  </select>
+        #
+        # @example Prompt option
+        #   <%=
+        #     # ...
+        #     values = Hash['it' => 'Italy', 'us' => 'United States']
+        #     select :stores, values, options: {prompt: 'Select a City'}
+        #   %>
+        #
+        #   # Output:
+        #   #  <select name="book[store]" id="book-store">
+        #   #    <option>Select a City</option>
+        #   #    <option value="it">Italy</option>
+        #   #    <option value="us">United States</option>
+        #   #  </select>
         def select(name, values, attributes = {})
-          options    = attributes.delete(:options) || {}
-          attributes = { name: _input_name(name), id: _input_id(name) }.merge(attributes)
+          options      = attributes.delete(:options) { {} }
+          attributes   = { name: _input_name(name), id: _input_id(name) }.merge(attributes)
+          blank_value  = options.delete(:include_blank)
```

Comment:
@davydovanton `include_blank: true` overlaps with `prompt: ""`. What if we remove the first? :wink: